src/main/java/com/swervedrivespecialties/exampleswerve/RobotContainer.java

```diff
@@ -0,0 +1,70 @@
+package com.swervedrivespecialties.autonrefresh;
+
+import com.swervedrivespecialties.autonrefresh.subsystems.DrivetrainSubsystem;
+import com.swervedrivespecialties.autonrefresh.util.AutonChooser;
+import com.swervedrivespecialties.autonrefresh.util.BeakXboxController;
+import com.swervedrivespecialties.autonrefresh.util.util;
+import com.swervedrivespecialties.exampleswerve.commands.DriveCommand;
+
+public class RobotContainer {
+
+    public DrivetrainSubsystem drive = new DrivetrainSubsystem.getinstance(); 
+
+    private BeakXboxController controller1 = new BeakXboxController(RobotMap.PRIMARY_CONTROLLER_USB_PORT);
```

Comment:
```suggestion
```
All this code is technically correct, and will work in time. But you have to write BeakXBoxController before you can use it